packages/esm-service-queues-app/src/queue-patient-linelists/queue-linelist-filter.test.tsx

```diff
@@ -15,6 +16,25 @@ const workspaceProps = {
   setTitle: jest.fn(),
 };
 
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  OpenmrsDatePicker: jest.fn(({ id, labelText, value, onChange }) => (
```

Comment:
It would be better if we didn't have to mock this component out at all. It shouldn't need to be, since I don't think it depends on any state other than the props passed in.